authui/src/authflowv2/components/select.css

```diff
@@ -18,14 +18,45 @@
 
     --select__search-border: 1px solid var(--color-grey-100);
 
-    --select__input-padding: 0.75em 1em 0.75em 3.25em;
+    --select__search-input-padding: 0.875em 0.625em;
+    @media (max-width: theme("screens.tablet")) {
+      --select__search-input-padding: 0.875em 0.125em;
+    }
```

Comment:
According to our discussion yesterday, it should be two variables instead of media query?
```
--select__search-input-padding--mobile
--select__search-input-padding--desktop
```

Reply:
Addressed in https://github.com/authgear/authgear-server/pull/3669/commits/2488054563bfea4ce7410ecace52c1618ca65498